cg/cli/workflow/balsamic/base.py

```diff
@@ -197,28 +197,26 @@ def start(
     dry_run: bool,
 ):
     """Start full workflow for case ID."""
+    analysis_api: BalsamicAnalysisAPI = context.obj.meta_apis["analysis_api"]
+    analysis_api.prepare_fastq_files(case_id=case_id, dry_run=dry_run)
```

Comment:
Since this function call is made for all flows, can it be moved into a base command instead?

Reply:
Hm, did not find a great way to do it - this might be the best way

Reply:
I think this is potentially out of scope for this PR, although I like the idea. What do you think @Vince-janv?

Reply:
I'd also like to have all function calls in one place. The question that I find hard to answer is how to put it somewhere that is intuitive, so that we're not hiding code from people 🤔

Reply:
As in "it does not work setting it in CLI/workflow/base.py" or as in "it would be hidden in CLI/workflow/base.py"?

Reply:
As in, I don't think a person looking at the start command would expect other function calls to have been made before (via the chain command thing going on). But if we implement the pattern people might learn it. I think it can be very useful

Reply:
Good point. I don't have a big issue with having a one-liner in the start of every "start" command. In this case I feel like the added benefit would be overruled by the introduced obscurity.

Reply:
Although I guess we would not need to add that line for any potential future pipelines if it was moved.